infrastructure/modules/app_service/main.tf

```diff
@@ -0,0 +1,29 @@
+resource "azurerm_service_plan" "api_plan" {
+  name                = var.service_plan_name
+  resource_group_name = var.resource_group_name
+  location            = var.physical_location
+  os_type             = "Linux"
```

Comment:
Maybe we could add os_type as a variable? It could be nice if someday for some reason we want to go on an Windows based OS. Tell me if you agree